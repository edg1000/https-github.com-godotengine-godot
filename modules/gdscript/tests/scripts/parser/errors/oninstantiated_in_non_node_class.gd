@oninstantiated var node

func test():
	pass
