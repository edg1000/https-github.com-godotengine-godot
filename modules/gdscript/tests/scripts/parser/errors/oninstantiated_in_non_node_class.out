GDTEST_ANALYZER_ERROR
"@oninstantiated" can only be used in classes that inherit "Node".
